xbee_port: /dev/ttyUSB0
baud_rate: 115200
compressed: true
debug: False
#subscriber_list: [{node_name: 'broadcast', topic: /gps_box/imu/data, type: {package: sensor_msgs, name: Imu}}, {node_name: 'broadcast', topic: /gps_box/gps/fix, type: {package: sensor_msgs, name: NavSatFix}}]
